tests/integration/ovs_test.py

```diff
@@ -103,3 +117,34 @@ def test_create_and_remove_ovs_bridge_with_internal_port_and_static_ip():
 
     assertlib.assert_absent(BRIDGE1)
     assertlib.assert_absent(PORT1)
+
+
+@pytest.mark.xfail(raises=NmstateLibnmError, strict=True)
+def test_nm_ovs_plugin_missing():
+    with disable_nm_ovs_plugin():
+        libnmstate.apply(
+            {
+                Interface.KEY: [
+                    {
+                        Interface.NAME: BRIDGE1,
+                        Interface.TYPE: InterfaceType.OVS_BRIDGE,
+                        Interface.STATE: InterfaceState.UP,
+                    }
+                ]
+            }
+        )
+
+
+@contextmanager
+def disable_nm_ovs_plugin():
+    libcmd.exec_cmd(DNF_REMOVE_NM_OVS_CMD)
+    libcmd.exec_cmd(SYSTEMCTL_RESTART_NM_CMD)
+    ret = [1,0,0]
+    while ret[0]:
+        ret = libcmd.exec_cmd(SYSTEMCTL_IS_ACTIVE_NM_CMD)
+        time.sleep(1)
+    try:
+        yield
+    finally:
+        libcmd.exec_cmd(DNF_INSTALL_NM_OVS_CMD)
+        libcmd.exec_cmd(SYSTEMCTL_RESTART_NM_CMD)
```

Comment:
The error log is hard to understand, nevertheless. If this was code is mostly correct, the is-active check should run here as well. Can you check if this fixes it?

Reply:
Nope, not working..

Reply:
I took a deeper look. It seems that NM is actually working already but the dbus code in Nmstate needs to be re-initialized.